app/src/main/java/com/github/polybooks/core/Sales.kt

```diff
@@ -23,7 +23,7 @@ data class Sale(
         val seller : User,
         val price : Float,
         val condition : BookCondition,
-        val date : Timestamp,
+        val date : Timestamp?, // TODO remove nullable when timestamp serialization situation will be resolved
```

Comment:
Is it because of the nullable that the Sale are Serializable and that the tests pass? So now we are still using Firebase timestamp?

Reply:
Since it is nullable AND I instanciate a sale with `null` in the date, then the serialization is working ^^. But obviously it is a temporary fix that we should finally address